### PR TITLE
Feature/connector version api

### PR DESCRIPTION
**Pull request by wvanhaevre**
Currently supporting: Conviva, Nielsen and Comscore

### CODE REVIEW

File: THEOplayer-Connector-Conviva.podspec [hunk 1]
```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.0", :tvos => "13.0" }
 
   s.source_files = 'Code/Conviva/Source/**/*'
+  s.resource_bundles = {'THEOplayer-Connector-Conviva' => ['version.json']}
```
Comment (Danesz) [inline THEOplayer-Connector-Conviva.podspec:19]: I think we need to do the same inclusion for the SPM packages too.
(and I think when you access the resources through SPM vs Cocoapods, something need to be changed when looking for the path. I think @Dev1an had some issues with this in the past)

File: THEOplayer-Connector-Comscore.podspec [hunk 2]
```diff
@@ -16,9 +16,11 @@ Pod::Spec.new do |s|
...
 
   s.source_files = 'Code/Comscore/Source/**/*'
+  s.resource_bundles = {'THEOplayer-Connector-Comscore' => ['version.json']}
       
   s.static_framework = true
   s.swift_versions = ['5.3', '5.4', '5.5', '5.6', '5.7']
-  s.dependency 'THEOplayerSDK-core', "~> 8"
   s.dependency 'ComScore', '~> 6.10.0'
+  s.dependency 'THEOplayerSDK-core', "~> 8"
+  s.dependency 'THEOplayer-Connector-Utilities', "~> " + theoplayer_connector_major_minor_version, ">= " + theoplayer_connector_version
```
Comment (Danesz) [inline THEOplayer-Connector-Comscore.podspec:25]: there are some Utilities-specific checks in our GItHub action pipeline.
Were those already taken into account?

**Comment by Dev1an** [comment]
Hi @wvanhaevre,
Maybe I come a bit late to the game but why don't we use the same approach as we use for THEOplayer core? That is, the version accessible through a global variable `THEOplayerVersion`.

That way we cannot have any runtime failures to look up the version (for example due to SPM vs Cocoapods differences).

**Comment by wvanhaevre** [comment]
> Hi @wvanhaevre, Maybe I come a bit late to the game but why don't we use the same approach as we use for THEOplayer core? That is, the version accessible through a global variable `THEOplayerVersion`.
> 
> That way we cannot have any runtime failures to look up the version (for example due to SPM vs Cocoapods differences).

Would be ok for me, but on the core this is a value that requires a manual update for every release. It felt like an easy source for mistakes. Certainly when knowing that for every SDK release we  currently also release all connectors.

**Comment by Dev1an** [comment]
For Core this is done in a release script I think. (see `scripts/node/src/release-tool.ts` line `235`)

**Comment by wvanhaevre** [comment]
To be reworked to also support SPM

**Pull request closed without merging**